onebusaway-transit-data-federation/src/main/java/org/onebusaway/transit_data_federation/impl/realtime/gtfs_realtime/GtfsRealtimeSource.java

```diff
@@ -366,7 +375,21 @@ private FeedMessage readFeedFromUrl(URL url) throws IOException {
       }
     }
   }
-
+/**
+ * Set the headers to the urlConnection if any
+ * @param urlConnection
+ * @return, the urlConnection with the headers set
+ */
+  private void setHeadersToUrlConnection(URLConnection urlConnection)
+  {
+	  if(_headersMap!= null)
```

Comment:
@barbeau <https://github.com/barbeau> Good eye for the "!=". I will make
the changes tonight.

And sorry for the braces, C# habit

2015-03-17 12:08 GMT-04:00 Sean Barbeau <notifications@github.com>:

> In
> onebusaway-transit-data-federation/src/main/java/org/onebusaway/transit_data_federation/impl/realtime/gtfs_realtime/GtfsRealtimeSource.java
> <https://github.com/OneBusAway/onebusaway-application-modules/pull/125#discussion_r26587101>
> :
>
> > @@ -366,7 +375,21 @@ private FeedMessage readFeedFromUrl(URL url) throws IOException {
> >        }
> >      }
> >    }
> > -
> > +/**
> > + * Set the headers to the urlConnection if any
> > + * @param urlConnection
> > + * @return, the urlConnection with the headers set
> > + */
> > +  private void setHeadersToUrlConnection(URLConnection urlConnection)
> > +  {
> > +	  if(_headersMap!= null)
>
> Actually, braces should be shifted too to match style. I was looking at
> this on mobile and didn't see that at first.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/OneBusAway/onebusaway-application-modules/pull/125/files#r26587101>
> .
>